integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java

```diff
@@ -130,15 +120,16 @@
  * Tests related to Domain events logged by operator.
  * The tests checks for the following events in the domain name space.
  * DomainCreated, DomainChanged, DomainDeleted, DomainCompleted,
- * DomainFailed, NamespaceWatchingStarted, and NamespaceWatchingStopped.
+ * DomainFailed.
```

Comment:
In the above names, the prefix is removed. Please change the events to Created, Changed, Deleted, Completed, Failed

Reply:
and throughout this class

Reply:
done